Dockerfile.cli

```diff
@@ -1,11 +1,14 @@
 FROM python:3.9-slim as base
 
 RUN apt-get update && \
-    apt-get install -y git make gcc python3-dev && \
+    apt-get install -y git git-lfs python3-dev && \
```

Comment:
We should pin this to a version of git-lfs that works. 3.1.x definitely doesn't, 2.12.1 does (it's what our envs are pinned to)

Reply:
good point, thanks for the reminder

Reply:
so the installed version is 

```
git lfs --version
git-lfs/2.13.2 
```

I guess it's ok if I pin to this?

Reply:
Yes I think <3.x is ok